site/index.html

```diff
@@ -80,7 +82,9 @@ <h3>Official Themes</h3>
     <p>
       Take Bootstrap 4 to the next level with official premium themes—toolkits built on Bootstrap with new components and plugins, docs, and build tools.
     </p>
-    <img class="img-fluid mt-3 mx-auto" src="{{ site.baseurl }}/docs/{{ site.docs_version }}/assets/img/bootstrap-themes.png" alt="Bootstrap Themes" width="1024" height="388">
```

Comment:
I think we can save some bandwidth if we use a `.jpg` here, the file size of complex images tend to be higher with `.png`

Reply:
Last time I tried, we didn't save a lot with JPG 80%.

Reply:
@MartijnCuppens: I'm gonna merge this as is for now, but feel free to experiment with JPG for the themes image and example images later.